login-workflow/src/new-architecture/screens/AccountDetailsScreen/AccountDetailsScreen.tsx

```diff
@@ -20,15 +20,7 @@ export const AccountDetailsScreen: React.FC<AccountDetailsScreenProps> = (props)
         try {
             setIsLoading(true);
             await actions().setAccountDetails({ firstName, lastName });
-            if (currentScreen === totalScreens - 2) {
-                const { email, organizationName } = await actions().completeRegistration(
-                    { firstName, lastName },
-                    screenData.VerifyCode.code,
-                    screenData.CreateAccount.emailAddress
-                );
-                updateScreenData({ screenId: 'RegistrationSuccessScreen', values: { email, organizationName } });
-            }
-            nextScreen({
+            await nextScreen({
```

Comment:
Can you double-check your types? In this case, you are awaiting a promise for `nextScreen` but using void in other locations.